README.md

```diff
@@ -11,31 +11,33 @@ ___
 * [Set-up](#set-up)
 * [How to use](#how-to-use)
 * [Exposed API](#exposed-api)
-	* [Auth](#auth)
-	* [SPTAudioStreamingController Class](#sptaudiostreamingcontroller-class)
-	* [SPTSearch Class](#sptsearch-class)
+  * [Auth](#auth)
+  * [SPTAudioStreamingController Class](#sptaudiostreamingcontroller-class)
+  * [SPTSearch Class](#sptsearch-class)
 * [Demo](#demo) 
 
 ___
 
 ## Set-up:
+
 ### Using npm
->Recomended
+
+>Recommended
 
 1. Use `npm install react-native-spotify` from the directory of your project in your command line.
 
-2. Download the Spotify IOS SDK beta 17 [here](https://github.com/spotify/ios-sdk/releases/tag/beta-17) and unzip it.
+2. Download the Spotify IOS SDK beta 25 [here](https://github.com/spotify/ios-sdk/releases/tag/beta-25) and unzip it, you will find 3 files with the `.framework` extension.
 
-3. Open your react native project in Xcode and drag the unzipped `Spotify.framework` file into the `Frameworks` group in your Xcode project (create the group if it doesn’t already exist). In the import dialog, tick the box for **Copy items into destinations group folder** (or **Destination: Copy items if needed**).
+3. Open your react native project in Xcode and drag the 3 unzipped `.framework` files (`SpotifyAudioPlayback`, `SpotifyAuthentication`, `SpotifyMetadata`) into the `Frameworks` group in your Xcode project (create the group if it doesn’t already exist). In the import dialog, tick the box for **Copy items into destinations group folder** (or **Destination: Copy items if needed**).
 
-4. Please folow the instructions on the **"Creating Your Client ID, Secret and Callback URI"** and **"Setting Up Your Build Environment"** sections of the [*Spotify iOS SDK Tutorial*](https://developer.spotify.com/technologies/spotify-ios-sdk/tutorial/) 
->**Important Note!** When adding frameworks to the list in the "**Link Binary With Libraries**" section you wll need to add `WebKit.framework` in addition to those mentioned in the tutorial.
+4. Please follow the instructions on the **"Creating Your Client ID, Secret and Callback URI"** and **"Setting Up Your Build Environment"** sections of the [*Spotify iOS SDK Tutorial*](https://developer.spotify.com/technologies/spotify-ios-sdk/tutorial/) 
+>**Important Note!** When adding frameworks to the list in the "**Link Binary With Libraries**" section you will need to add `WebKit.framework` in addition to those mentioned in the tutorial.
 
 5. Go to `node-modules/react-native-spotify/` and copy the following files to the `ios` directory of your project:
-	* `SpotifyLoginViewController.m`
-	* `SpotifyLoginViewController.h`
-	* `SpotifyAuth.m`
-	* `SpotifyAuth.h`
+  * `SpotifyLoginViewController.m`
+  * `SpotifyLoginViewController.h`
+  * `SpotifyAuth.m`
+  * `SpotifyAuth.h`
```

Comment:
It might be helpful to add that it's required to add these references to `AppDelegate.m`

```#import <SpotifyAuthentication/SpotifyAuthentication.h>
#import <SpotifyMetadata/SpotifyMetadata.h>
#import <SpotifyAudioPlayback/SpotifyAudioPlayback.h>
```